app/views/govuk_publishing_components/components/docs/image_card.yml

```diff
@@ -205,6 +205,47 @@ examples:
       <div class="govuk-!-width-full">
         <%= component %>
       </div>
+  youtube_video_with_brand:
+    data:
+      brand: "department-for-work-pensions"
+      youtube_video_id: "XnP6jaK7ZAY"
+      href: "/still-not-a-page"
+      image_src: "https://assets.publishing.service.gov.uk/government/uploads/system/uploads/feature/image/91397/s712_SG_Swear_in_1_.jpg"
+      image_alt: "The vision for GOV.UK"
```

Comment:
For these examples I think `image_alt` is used but am I right in thinking `image_src` is ignored and could be removed?